guides/source/active_record_basics.md

```diff
@@ -493,7 +493,7 @@ book = Book.find(42)
 The above resulting in this SQL:
 
 ```sql
-SELECT "books".* FROM "books" WHERE "books"."author" = ? LIMIT ?  [["author", "J.R.R. Tolkien"], ["LIMIT", 1]]
+SELECT "books".* FROM "books" WHERE "books"."title" = ? LIMIT ?  [["title", "Metaprogramming Ruby 2"], ["LIMIT", 1]]
```

Comment:
While we are changing this...
I think it would be nice to add comments describing the call, similar to the example above it:
```suggestion
-- Book.find_by(title: "Metaprogramming Ruby 2")
SELECT "books".* FROM "books" WHERE "books"."title" = ? LIMIT ?  [["title", "Metaprogramming Ruby 2"], ["LIMIT", 1]]
```